lib/muontrap/daemon.ex

```diff
@@ -58,6 +58,7 @@ defmodule MuonTrap.Daemon do
     :command,
     :port,
     :cgroup_path,
+    :custom_logger,
```

Comment:
Good name, bad name? Maybe `:logger` or `:logger_fun` would be better instead?

Reply:
I like `:logger_fun` since it makes it clear (at least to me) that you're supposed to give it a function. I'd probably look up in the docs for the other names just in case I was supposed to pass a behaviour.